### ISSUE TITLE
automate the aggregation of the toplevel NEWS file from subproject NEWS files

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3199]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#3199|
|Type|enhancement|
|Created|2008-04-20 22:17:30Z|
|Branch|https://github.com/twisted/twisted/tree/aggregate-news-3199|

There are subproject NEWS files in twisted/*/topfiles/NEWS, and they are manually aggregated into a toplevel NEWS file for every release. This should be automated.

This is related to [#2884](https://github.com/twisted/twisted/issues/2884), but much simpler.

<details><summary>Searchable metadata</summary>

```
trac-id__3199 3199
type__enhancement enhancement
reporter__radix radix
priority__normal normal
milestone__totally_automated_release_infrastructure totally automated release infrastructure
branch__branches_aggregate_news_3199 branches/aggregate-news-3199
branch_author__radix__thijs radix, thijs
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1208729850000000 1208729850000000
changetime__1259933661000000 1259933661000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_1"></a>|@radix commented|
|-|-|

(In [26257]) Branching to 'aggregate-news-3199'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set status to `closed`|
|-|-|

In [27678]: Merge news-generator-2884-3 Author: exarkun Reviewer: glyph Fixes: [#2884](https://github.com/twisted/twisted/issues/2884) Fixes: [#3199](https://github.com/twisted/twisted/issues/3199) Add an administrative tool for generating news files for Twisted and Twisted subproject releases. This tool uses the files now required to be added to various `topfiles` directories when resolving tickets.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
